src/data/thunks.js

```diff
@@ -168,7 +175,7 @@ export function stopExam() {
   };
 }
 
-export function continueExam() {
+export function continueExam(noLoading = true) {
```

Comment:
I think this is mainly used when you are on ready_to_submit page and instead of clicking 'submit' you click 'continue my exam', then this function gets called (this is also how it is done in [LMS](https://github.com/edx/edx-proctoring/blob/ac15e68b4ae3afd8c4fe29edd32697f095ba84fa/edx_proctoring/templates/timed_exam/ready_to_submit.html#L24))

Also in this PR this function is used to start proctored exam (I'm not sure why I did this) but it gets refactored in #14